package/src/components/AttachmentPicker/AttachmentPicker.tsx

```diff
@@ -181,8 +181,14 @@ export const AttachmentPicker = React.forwardRef(
     }, [selectedPicker, closePicker]);
 
     useEffect(() => {
+      const onKeyboardOpenHandler = () => {
+        if (selectedPicker) {
+          setSelectedPicker(undefined);
+        }
```

Comment:
can this be done like this 

setSelectedPicker(undefined)

there is no need for an if at all.. if you want it to set it to undefined whenever keyboard is open

Reply:
Ideally, it won't create much of a difference. Its just a sanity check that we set it as undefined only when there is a value in the selectedPicker state. The same is done for the back action handler in the file.